packages/server/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM node:12-alpine
+FROM node:12-slim
```

Comment:
The oracle client relies on several system binaries that are not present in alpine linux. Even when added manually there are still issues and inconsistencies with the regular linux install script, while creating an image almost the same size. The cleanest approach was to bump the node base image to slim. 

This brings the total size of the base image from `89.5MB` to `142MB`.
Coupled with the installation of the instant client this brings the total image size of the server  from `1.22GB` to `1.45GB`

Reply:
All good - to be honest our app server image could probably be smaller in general. It does quite a lot. I saw this the other day, which might be worth a try at some point to see if it can reduce our images down. It would also make deployments etc much faster.

https://github.com/docker-slim/docker-slim